es/index.js

```diff
@@ -154,7 +169,8 @@ export default function axiosRetry(axios, defaultOptions) {
 
     const {
       retries = 3,
-      retryCondition = isNetworkOrIdempotentRequestError
+      retryCondition = isNetworkOrIdempotentRequestError,
+      delayStrategy = noDelay
```

Comment:
The new `delayStrategy` option should be added to the jsdocs for this function

Reply:
Rookie mistake! Thanks for making me aware.